ONSdigital/dp-frontend-filter-dataset-controller
# Add a healthcheck to the filter-dataset-controller

### What

- Add a healthcheck to the filter-dataset-controller
- Service checks health of external services every 60 secs

### How to review

- Call the /healthcheck endpoint with the service running
- Note you should only get InternalServerError at this stage due to other services not yet containing healthchecks
- Verify that errors are recorded in service logs

### Who can review

- Anyone
